Mathlib/Order/ConditionallyCompleteLattice/Group.lean

```diff
@@ -19,34 +19,34 @@ variable {α : Type*} {ι : Sort*} {ι' : Sort*} [Nonempty ι] [Nonempty ι']
   [ConditionallyCompleteLattice α] [Group α]
 
 @[to_additive]
-theorem le_mul_ciInf [CovariantClass α α (· * ·) (· ≤ ·)] {a : α} {g : α} {h : ι → α}
+theorem le_mul_ciInf [MulLeftMono α] {a : α} {g : α} {h : ι → α}
     (H : ∀ j, a ≤ g * h j) : a ≤ g * iInf h :=
   inv_mul_le_iff_le_mul.mp <| le_ciInf fun _ => inv_mul_le_iff_le_mul.mpr <| H _
 
 @[to_additive]
-theorem mul_ciSup_le [CovariantClass α α (· * ·) (· ≤ ·)] {a : α} {g : α} {h : ι → α}
+theorem mul_ciSup_le [MulLeftMono α] {a : α} {g : α} {h : ι → α}
     (H : ∀ j, g * h j ≤ a) : g * iSup h ≤ a :=
   le_mul_ciInf (α := αᵒᵈ) H
 
 @[to_additive]
-theorem le_ciInf_mul [CovariantClass α α (Function.swap (· * ·)) (· ≤ ·)] {a : α} {g : ι → α}
+theorem le_ciInf_mul [MulRightMono α] {a : α} {g : ι → α}
     {h : α} (H : ∀ i, a ≤ g i * h) : a ≤ iInf g * h :=
   mul_inv_le_iff_le_mul.mp <| le_ciInf fun _ => mul_inv_le_iff_le_mul.mpr <| H _
 
 @[to_additive]
-theorem ciSup_mul_le [CovariantClass α α (Function.swap (· * ·)) (· ≤ ·)] {a : α} {g : ι → α}
+theorem ciSup_mul_le [MulRightMono α] {a : α} {g : ι → α}
     {h : α} (H : ∀ i, g i * h ≤ a) : iSup g * h ≤ a :=
   le_ciInf_mul (α := αᵒᵈ) H
 
 @[to_additive]
-theorem le_ciInf_mul_ciInf [CovariantClass α α (· * ·) (· ≤ ·)]
-    [CovariantClass α α (Function.swap (· * ·)) (· ≤ ·)] {a : α} {g : ι → α} {h : ι' → α}
+theorem le_ciInf_mul_ciInf [MulLeftMono α]
```

Comment:
I think it'd look nicer if the typeclass arguments were together in this line, but that's more subjective. Same below.